dt-core/admin/menu/tabs/tab-workflows.php

```diff
@@ -275,9 +275,17 @@ public function content( $tab ) {
 
             $this->template( 'begin' );
 
+            ?>
+            <div id="post-body-content">
+            <?php
```

Comment:
This is creating a duplicate div. `$this->template('begin')` already adds this, so this can be removed.